unfetter-discover/unfetter-ui
# Removed unnecessary call to markForCheck()

Fixes unfetter-discover/unfetter#1511

No need to call markForCheck - marking defs are populated and displayed correctly, and are applied to the assessment and baseline correctly on save.

### To test:
1. Pull this branch
2. For assessments and baselines, do the following:
- Create a new one, skip directly to the Summary page of the wizard
- Select one or more marking definitions
- Verify there are no exceptions
- Click Save and verify that the selected marking definitions are displayed just under the assessment/baseline name in the top left corner
- Edit the assessment/baseline and make sure the selected marking definitions are populated on the Summary page
- Change the selection and Save
- Verify the updated marking definitions are displayed correctly
